build.py

```diff
@@ -119,26 +117,61 @@ def run(config, start_class_path, arguments):
   command += [ start_class_path ]
   command += arguments
 
-  print('running : %s' % command)
-  print('Build %s' % ('PASSED' if subprocess.call(command) == 0 else 'FAILED'))
-### def run ###
+  print 'Running: [',
+  for x in command :
+    print x,
+  print ']'
+  print('Run %s' % ('PASSED' if subprocess.call(command) == 0 else 'FAILED'))
+
+
+# USAGE
+#
+# Print basic usage info.
+#
+def usage() :
+  print('Usage: python build.py clean | build | rebuild | run | help')
+  print('  clean   : Remove all files in the output directory.')
+  print('            This does not remove the root of the output tree.')
+  print('  build   : Build the full project. This will build all java files')
+  print('            found in all of the src directories.')
+  print('  rebuild : perform clean followed by build.')
+  print('  run <class path> [ arguments ... ] : Run the specified class.')
+  print('            All arguments after the class path will be passed to')
+  print('            the java class when it runs.')
+  print('  help    : Print this helpful message.')
 
 
 # MAIN
 def main(args) :
-  if 'clean' == args[0] :
-    clean(CONFIG)
-  elif 'build' == args[0] :
-    build(CONFIG)
-  elif 'rebuild' == args[0] :
-    clean(CONFIG)
-    build(CONFIG)
-  elif 'run' == args[0] :
-    run(CONFIG, args[1], args[2:])
+  if len(args) > 1 :
```

Comment:
I think this branching structure is hard to read. Is there a way we can make it easier to read.